srv/modules/runners/deepsea.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+"""
+This runner implements helper functions for DeepSea in general
+"""
+
+DEEPSEA_VERSION = 'DEVVERSION'
+
+
+def version(**kwargs):
+    # pylint: disable=unused-argument
+    """
+    Returns the DeepSea version info currently installed
+    """
+    return DEEPSEA_VERSION
```

Comment:
The response of this function should return a dictionary with the format similar to https://github.com/SUSE/DeepSea/compare/master...rjfd:wip-version-number2?expand=1#diff-282d3ea0e3fce515fa2624db3082c383R11 ,
which was the response agreed with the openATTIC team.

Reply:
I was not aware that such an agreement exists. Was this not shared or did I miss this somehow?

Reply:
I reached out to you on IRC.  This is the last pending PR and I need it done.

Reply:
There was already an implementation ready in https://github.com/SUSE/DeepSea/compare/master...rjfd:wip-version-number2?expand=1
which implements the same thing as this one, and was waiting for your feedback.

Reply:
It's a branch, not a PR.  I reached out to you on IRC.  I have others waiting on me and this PR satisfies the minimum requirements.

Reply:
Yes it was a branch, but I thought that it was agreed that as soon as you gave your green light on it I would push the commits to the currently opened PR #817 .

Since this might got misunderstood I already pushed the implementation based on git annotated tags to PR #817. Therefore, I believe you can close this PR.